walkingastro.lic

```diff
@@ -1,3 +1,7 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#walkingastro
```

Comment:
💡 It's a personal preference, but I prefer to break up words when possible, e.g. `walking-astro`